src/main/java/com/questhelper/steps/ObjectStep.java

```diff
@@ -100,7 +101,7 @@ public ObjectStep(QuestHelper questHelper, int objectID, WorldPoint worldPoint,
 
 	public ObjectStep copy()
 	{
-		ObjectStep newStep = new ObjectStep(getQuestHelper(), objectID, worldPoint, null, requirements, recommended);
+		ObjectStep newStep = new ObjectStep(getQuestHelper(), objectID, definedPoint.getWorldPoint(), null, requirements, recommended);
```

Comment:
With all these; how clean is it to unwrap and then immediately re-wrap the points?